src/components/constants/constants.js

```diff
@@ -0,0 +1,44 @@
+export const pokeApiUrl = "https://pokeapi.co/api/v2/pokemon?limit=25&offset=0"
+
+export const pokeType = (pokemonType) => {
```

Comment:
Let's convert to a switch case and transfer to `utils.ts` instead of placing here in constants since it's a util function.

Reply:
Rename the function to make it more descriptive too. Example would be `getClassFromPokeType`.